pkg/operator/api/api.go

```diff
@@ -32,13 +32,14 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/client"
 
 	commonapi "github.com/dapr/dapr/pkg/apis/common"
-	componentsapi "github.com/dapr/dapr/pkg/apis/components/v1alpha1"
+	compapi "github.com/dapr/dapr/pkg/apis/components/v1alpha1"
```

Comment:
This change make codes misunderstand.

Reply:
`compapi` was chosen because it is a smaller string than `componentsapi` to follow best practice of not having over verbose variable names.